wingpio/gpioapi.cpp

```diff
@@ -116,14 +118,10 @@ extern "C" {
                 case GpioPinDriveMode::OutputOpenDrainPullUp:
                     if (value == PINVALUE_HIGH) {
                         pin->Write(GpioPinValue::High);
-                        ret = SUCCESS;
                     } else if (value == PINVALUE_LOW) {
                         pin->Write(GpioPinValue::Low);
-                        ret = SUCCESS;
-                    } else {
-                        PyErr_Format(PyExc_TypeError, "Invalid pin value specified for pin %d", pin->PinNumber);
                     }
```

Comment:
We are removing this b/c it doesn't follow RPi.GPIO's behavior.  The 'value' in this case is an optional parameter.  If the 'value' is neither HIGH/LOW, RPi.GPIO ignores it.   I found that recently while I was using raspbian to look at another issue.